resources/test-config.edn

```diff
@@ -6,8 +6,9 @@
  :peer-config
  {:zookeeper/address "127.0.0.1:2188"
   :onyx.peer/job-scheduler :onyx.job-scheduler/greedy
-  :onyx.messaging/impl :aeron
-  :onyx.messaging/ip "localhost"}
+  :onyx.messaging/config {:messaging/impl :aeron
```

Comment:
I'd prefer to keep the configuration flat for two reasons. 1 - namespacing at the innermost level (`messaging/impl`) starts to lose the essence of why it's namespaced to begin with. 2 - flat configurations are easier to drop into databases. They require less mangling for the user. Is there something particularly annoying about flat configurations in your usage?